.github/CONTRIBUTING.md

```diff
@@ -540,7 +540,113 @@ For more on good git commits, see (among _many_ other):
 
 ### Pull-request
 
-* How is a pull-request that is easily reviewed constructed?
+Why should you care about creating good pull-requests - is it not just a tool for code reviews?
+
+Well first of all, a good pull-request can help reduce how long it takes for your changes to be merged to master by making the life easier for the reviewer, causing a quicker review. 
+
+Secondly a good pull-request is more than just a tool for code reviews - it documents the process of reaching a satisfactory implementation aswell as toughts and discussion surrounding this. 
+
+The rules that applies for writing a [good issue](#issue) also applies for writing a good description for your pull-request.
+But more importantly a pull-request includes commits containing code changes that you want to get merged to master, which means there is a few more points to keep in mind, when constructing a good pull-request. 
+
+Before going through these however - we want to remind you to take ownership of your pull-requests. 
+It is your responsibility as a contributor to get your pull-requests reviewed, therefore reach out and be communicative! 
+
+### The good pull-request solves one issue
+
+Every pull-request that is created should solve _one_ existing [Github issue](https://github.com/kirbydesign/designsystem/issues). 
+
+There might be cases where the changes in a pull-request naturally solves multiple issues, often caused by them being related one way or another. This is the exception, not the rule. 
+Striving for the scope of the pull-requests to be one issue makes the task of reviewing it more manageable.
+
+If you are not able to reference any issue when filling in the `This PR closes # (reference issue number here)` part of the pull-request template or when following the branch name convention outlined in [The Good: Branch](#branch); it is unfortunately an indicator that you are not solving any issue and have not followed [the process for contribution](#the-process-for-contribution). 
```

Comment:
Consider rephrasing this paragraph to move the point up front. Maybe move the examples to a sentence after it:

> If you are not able to reference ~~any~~ a particular/specific issue ~~when filling in the `This PR closes # (reference issue number here)` part of the pull-request template or when following the branch name convention outlined in [The Good: Branch](#branch);~~ it is unfortunately an indicator that you are not solving any issue and have not followed [the process for contribution](#the-process-for-contribution). [put the _when filling in the `This PR closes # (reference issue number here)` part of the pull-request template or when following the branch name convention outlined in [The Good: Branch](#branch);_ part here]

Does that make sense?

Reply:
Good point! 
I've rephrased it to: 
>If you are not able to reference any specific issue; it is unfortunately an indicator that you have not followed [the process for contribution](#the-process-for-contribution). For example when filling in the `This PR closes # (reference issue number here)` part of the pull request or when following the name convention outlined in [The Good: Branch](#branch).